execution_engine_testing/tests/bin/disk_use.rs

```diff
@@ -0,0 +1,192 @@
+use std::{
```

Comment:
Could we have a brief README.md for this new binary please?  Similar to the one in `profiling`: just some info about what the binary is for, how to run it, how to interpret the results, etc.

Reply:
Added to the README.md of execution_engine_testing.